css/styles.css

```diff
@@ -0,0 +1,66 @@
+* {
+  box-sizing: border-box;
+}
+
+form {
+  background-color: rgb(243, 243, 243);
+  padding: 0 1.5rem;
+}
+
+.column-full {
+  padding: 0;
+}
+
+h3 {
+  background-color: rgb(93, 25, 110);
+  color: white;
+  margin: 0;
+  padding: 10px;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-weight: 100;
+}
+
+h1 {
+  margin: 5px 10px;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+
+img {
+  min-width: 100%;
+  height: 300px;
+  object-fit: cover;
+}
```

Comment:
for this just set the img to `width: 100%` it's better since this is a half column, for the image to take up 100% of it's container at all times (it also preserves aspect ratio when you only size the width).